src/analyze/analyze.c

```diff
@@ -2385,10 +2396,17 @@ static int parse_argv(int argc, char *argv[]) {
                 return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
                                        "Option --root is only supported for cat-config and verify right now.");
```

Comment:
just one more change: the commit that adds --image should also modify this log message and the check above - ie, something like:

```
        if ((arg_root || arg_image) && !STRPTR_IN_SET(argv[optind], "cat-config", "verify"))
                return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
                                       "Options --root/--image are only supported for cat-config and verify right now.");
```